### ISSUE TITLE
"Too many scrolling linedefs" error not valid with v1.2 compat

### ISSUE THREAD

**Issue by tpoppins** [chocolate-doom/chocolate-doom#1445]
Levels with more than 64 scrolling linedefs cause Choco to exit with a "Too many scrolling linedefs" error, regardless of _-gameversion_.
This behavior is incorrect when _-gameversion 1.2_ is used because vanilla v1.2 did not have that limit. 

Two 1994 WADs that work without problems with v1.2:
[OPEN_AIR](https://www.doomworld.com/idgames/levels/doom/deathmatch/m-o/open_air)
[GOREFEST](https://www.doomworld.com/idgames/levels/doom/Ports/g-i/gorefest)

**Comment by AXDOOMER** [comment]
Version 1.2 did not have a limit? Can you give more information on that? Maybe 1.2 still used a static array but had no error check?

**Comment by tpoppins** [comment]
> Maybe 1.2 still used a static array but had no error check?

Yes, I think this is the case.

Did some experimenting: ran v1.2 on a test map, gradually increasing the number of linedefs assigned special 48. 
No problems with up to 66; with 67 scrolling linedefs the framerate takes an abrupt nosedive (20 dots with -devparm). At 70+ all player input is virtually ignored, though F10+Y still exits the game, albeit with a significant delay. With 900+ scrolling walls v1.2 freezes on startup.

**Comment by fabiangreffrath** [comment]
So, we could reasonably increase the limit to 66 for gameversion 1.2? Would this help to run the maps that you posted?

**Comment by tpoppins** [comment]
Yes, with MAXLINEANIMS raised to 66 both WADs run without errors. 

Each features exactly 66 scrolling linedefs. It's quite possible that the WADs' authors arrived at the same  number empirically, after seeing v1.2 freak out with more than 66 scrollers.
